engine/common/follower/engine.go

```diff
@@ -117,122 +141,149 @@ func New(
 	}
 	e.con = con
 
+	e.ComponentManager = component.NewComponentManagerBuilder().
+		AddWorker(e.processBlocksLoop).
+		Build()
+
 	return e, nil
 }
 
-// Ready returns a ready channel that is closed once the engine has fully
-// started. For consensus engine, this is true once the underlying consensus
-// algorithm has started.
+// Start starts follower loop, then the follower engine worker threads.
+func (e *Engine) Start(signalerContext irrecoverable.SignalerContext) {
+	e.log.Info().Msg("starting follower loop")
+	e.follower.Start(signalerContext)
```

Comment:
It would be nice to start the `FollowerLoop` separately here, like we plan to do for compliance engine and `EventLoop` in https://github.com/dapperlabs/flow-go/issues/6420 (added a note to that issue)

Reply:
I agree, thanks for noting it.